.changeset/lucky-hairs-tell.md

```diff
@@ -0,0 +1,12 @@
+---
+"@comet/blocks-admin": minor
+"@comet/blocks-api": minor
+"@comet/cms-site": minor
+---
+
+Add `VimeoVideoBlock` to support Vimeo videos
+
+The block is also added to the `VideoBlock`.
+
+
+
```

Comment:
```suggestion
```

This is a Demo-only change, so it's not needed in the changeset.

Reply:
[4c22d7d8d41d314f848f078348cab2c26150a982](https://github.com/vivid-planet/comet/pull/2068/commits/4c22d7d8d41d314f848f078348cab2c26150a982)